src/vivarium_census_prl_synth_pop/components/observers.py

```diff
@@ -37,7 +37,16 @@ def setup(self, builder: Builder):
         self.counter = 0
         self.output_path = self._build_output_root(builder) / "state_table.hdf"
         self.decennial_path = self._build_output_root(builder) / "decennial_census.hdf"
+
+        self.randomness = builder.randomness.get_stream(self.name)
         self.population_view = builder.population.get_view(columns=[])
+        response_probability_decennial = builder.lookup.build_table(
+            data=data_values.RESPONSE_PROBABILITY_DECENNIAL
+        )
+        self.response_probability_decennial = builder.value.register_value_producer(
```

Comment:
Do we expect this to change to a more complicated model where a simple lookup table wouldn't be sufficient?